### DISCUSSION TITLE
How to change the default list prefix?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#2985] [Q&A] [answered]
Hi,
I have tried with the settings below, but it does not work.

Is there a way to do it?

Thank you

```yml
crossref:
  lst-prefix: "sample"
```

**Comment by mcanouil** [comment, marked as answer]
Hi,

you can change both prefix (for crossref) and the title of the block you are referencing.

````
---
format: html
crossref:
  lst-prefix: "Something"
  lst-title: "Something"
---

```{#lst-customers .sql lst-cap="Customers Query"}
SELECT * FROM Customers
```

Then we query the customers database (@lst-customers).
````

**Reply by aborruso** [reply]
thank you

**Reply by aborruso** [reply]
@mcanouil there is nothing about `lst-title` in the documentation.

**Reply by mcanouil** [reply]
Indeed, but there is <https://quarto.org/docs/authoring/cross-references.html#titles> and <https://quarto.org/docs/authoring/cross-references.html#references-1>.
I agree this could be improved.
